### PR TITLE
Updated page layout and minor fixes. (#231)

### PR DESCRIPTION
**Pull request by alexdigdir**
CleanShot 2024-02-21 at 09 37 17

CleanShot 2024-02-21 at 09 37 45

### CODE REVIEW

**Review by TheKnarf — DISMISSED** [review]
🚀

File: packages/frontend-design-poc/src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-        <Route element={<PageLayout />}>
-          <Route path="/" element={<HelloWorld />} />
-          <Route path="/inbox" element={<Inbox />} />
-          <Route path="/inbox/:id" element={<InboxItemPage />} />
-          <Route path="*" element={<PageNotFound />} />
-        </Route>
-      </Routes>
-    </main>
-  );
+	return (
```
Comment (seanes) [inline packages/frontend-design-poc/src/App.tsx:11]: Hva har skjedd med indenteringen din?

File: packages/frontend-design-poc/src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+		<main className={styles.app}>
```
Comment (seanes) [inline packages/frontend-design-poc/src/App.tsx:12]: I think we should keep PageLayout as top level Layout component.
Home on the other hand doesn't make sense to have above PayLayout, but as a separate Page / Route component (similar to inibox), mounted on /.

I.e.:

- PageLayout (contains header, main / outlet, footer, etc)
       -  / :  Replace HelloWorld with Home
       - /inbox: <Inbox />
       etc.

File: packages/frontend-design-poc/src/components/PageLayout/PageLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+interface LayoutProps {
+	header: React.ReactNode;
+	sidebar: React.ReactNode;
+	main: React.ReactNode;
+	footer: React.ReactNode;
+}
+
+export const PageLayout: React.FC<LayoutProps> = ({ header, sidebar, main, footer }) => {
+	return (
```
Comment (seanes) [inline packages/frontend-design-poc/src/components/PageLayout/PageLayout.tsx:12]: I think I prefer the components themselves, e.g Footer, being composed as header, aside, main and footer intself. I would like to test the components in isolation  for wcag, not being depending on PageLayout.

File: packages/frontend-design-poc/src/App.tsx (hunk 1 above)
Comment (alexdigdir) [reply packages/frontend-design-poc/src/App.tsx:11]: biome setup

File: packages/frontend-design-poc/src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Outlet } from 'react-router-dom';
+import React from 'react';
+import { PageLayout } from '../../components/PageLayout';
+import { Footer, Header, Sidebar } from '../../components';
+
+export const Home = () => {
+	const [companyName, setCompanyName] = React.useState<string | undefined>('Aker Solutions AS');
```
Comment (seanes) [inline packages/frontend-design-poc/src/pages/Home/Home.tsx:7]: Går fint å bruke en tom string også her, vel?

**Review by seanes — APPROVED** [review]

**Pull request merged**